src/main.rs

```diff
@@ -578,9 +592,16 @@ fn run(args: Cli) -> Result<i32> {
         let cookie_jar = cookie_jar.lock().unwrap();
         s.save_cookies(
             cookie_jar
-                .matches(&url)
-                .into_iter()
-                .map(|c| cookie_crate::Cookie::from(c.clone()))
+                .iter_unexpired()
+                .map(|c| {
```

Comment:
**Note to self:** change `session::save_cookies()` arg type to `cookie_store::Cookie` and move this logic to `session.rs`